sandbox-backend/src/main/java/com/exadel/sandbox/team5/dto/CompanyDto.java

```diff
@@ -14,5 +14,5 @@
 public class CompanyDto extends IdentifierDto {
 
     private String name;
-    private Set<LocationDto> locations;
+    private Set<LocationDto> countries;
```

Comment:
Current response:
```
{
  ...
  "countries": [
    {
       "country": { 
	     "id": 111,
		 "name": "aaa"
		 ...
	   }
    },
    {
       "country": { 
	     "id": 222,
		 "name": "bbb"
		 ...
	   }
    }
  ]
}
```
So I think that is better to return only what is required using for example CountryDto, DTO shouldn't contain entity objects. As a result we will receive the following:
```
{
  ...
  "countries": [
    {
      "id": 111,
	  "name": "aaa"
    },
    {
      "id": 222,
	  "name": "bbb"
    }
  ]
}
```
or using the same `LocationDto` with `contryId`, `countryName` fields
```
{
  ...
  "locations": [
    {
      "countryId": 111,
	  "countryName": "aaa"
    },
    {
      "countryId": 222,
	  "countryName": "bbb"
    }
  ]
}
```